src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -61,9 +61,262 @@
 #include <beast/cxx14/memory.h> // <memory>
 #include <boost/foreach.hpp>
 #include <tuple>
+#include <ripple/basics/TestSuite.h>
 
 namespace ripple {
 
+namespace paging {
+    void
+    convertBlobsToTxResult (
+        std::vector<TxResult>& to,
+        std::uint32_t ledger_index,
+        std::string const& status, // the sql Status column
+        Blob const& txBlob,
+        Blob const& meta ) {
+
+        Serializer rawTxn (txBlob);
+        Serializer rawMeta (meta);
+        SerializerIterator it (rawTxn);
+        STTx::pointer txn = std::make_shared<STTx> (it);
+        auto tr = std::make_shared<Transaction> (txn, Validate::NO);
+
+        // TODO:
+        // tr->setStatus (status[0]);
+        tr->setLedger (ledger_index);
```

Comment:
Transaction::transactionFromSQL would set the status, which was some enumeration, using  a switch on the first character of this string.